openmc/mgxs/mgxs.py

```diff
@@ -1998,7 +1998,7 @@ def tallies(self):
         """
 
         # Instantiate tallies if they do not exist
-        if super(NuScatterMatrixXS, self).tallies is None:
+        if super(ScatterMatrixXS, self).tallies is None:
```

Comment:
I agree that this would be fine, the reason I left it as is was simply to "safeguard" the private `_tallies` attribute such that it will only be referenced directly in a single location by the abstract `MGXS` class. If you'd like to see a move to direct references to `_tallies` in all of the `tallies` properties I'll switch to that instead.